scss/bitstyles/settings/_color-palette.scss

```diff
@@ -109,3 +109,5 @@ $color-palette: (
     $percentages: $color-mix-percentages
   ),
 ) !default;
+
+$default-key: '100' !default;
```

Comment:
Just wondering: doesn't this increase the complexity of setting up bitstyles in a project quite a bit? I could imagine, that for the special cases, where the keys don't follow the classic scheme, one would instead require to just use the second parameter.

The way I understand it, the function already explodes when the color and percentage pair does not exist. So we wouldn't run into the problem of us accidentally not specifying it for these projects.

Just a though, maybe I'm also not fully getting the problem this PR is supposed to fix.

Reply:
When setting up, there’s no extra complexity if you’re suing the default color palette, or another palette with the same naming of colors. This change makes it more convenient to use the `palette.get()` function if you have a different naming scheme.

This makes it easier to use naming like:

```scss
'gray': (
  'default': #999,
  'light': #ccc,
  'dark': #666,
)
```
…because you can get the 'default' color by default when using `palette.get('gray')`.

Without this change, the function will throw an error on `palette.get('gray')`, because it looks for the `100` key, which doesn’t exist.

As you say, projects can explicitly use `palette.get('gray', 'default')`, but the convenience seemed worth the extra variable. Does that make sense, and does it seem worth it to you?

Reply:
I see! Thanks for the explanation, the use case just wasn't obvious for me at first 💡 

Makes perfect sense! Nice addition to bitstyles 💖